### PR TITLE
add credit endpoints and link token variations for income_verification

### PR DESCRIPTION
**Pull request by rajeevmehrotra**
This PR contains two changes:
- adds Link Token variations for income_verification product (1 for bank income, 1 for payroll income). Note these are within a sub-folder of the link tokens folder (see image below)
- adds /credit/ endpoints and /user/create endpoint

image

### CODE REVIEW

File: Plaid API Endpoints Collection.json [hunk 1]
```diff
@@ -78,8 +143,8 @@
 									]
 								}
 							},
-							"_postman_previewlanguage": null,
-							"header": null,
```
Comment (rajeevmehrotra) [inline Plaid API Endpoints Collection.json:82]: I don't know why all these changes occurred, happy to roll these particular ones back

**Review by phoenixy1 — COMMENTED** [review]
Hokay so the weird thing about publishing the postman collection is that the main thing you need to do is actually update the button in the readme. You do that like this:

in the postman app, select the collection, then click "..."->Share->Via run in postman->Embed a static version->generate code->add an environment (pick sandbox, make sure you remove your sandbox keys and secret from the environment variables first)->markdown friendly->update button and then test the button. Assuming it worked, copy and paste the generated markdown into the readme file.

**Comment by rajeevmehrotra** [comment]
@phoenixy1 got it thanks! should I include the changes to the .json files as well? or only the button in the readme?

**Comment by phoenixy1** [comment]
> @phoenixy1 got it thanks! should I include the changes to the .json files as well? or only the button in the readme?

You should probably include both! Although I am sort of tempted to remove the json file altogether but that's really unrelated.

**Comment by rajeevmehrotra** [comment]
got it, think I generated the button correctly @phoenixy1 let me know if it doesn't look right

**Review by phoenixy1 — COMMENTED** [review]
Awesome. Looks like this all works but I do have a couple of suggestions to make it 🔥 

a) Add examples responses, similar to what the other endpoints have
b) Add a test script to /user/create. If in the test tab  for /user/create you do the following:

let response = pm.response.json();
userToken = response.user_token;
pm.environment.set("user_token", userToken);

Then it will automatically populate the {{user_token}} variable used for the other Income endpoints so people don't have to copy and paste

**Comment by rajeevmehrotra** [comment]
cool think that should do it!

**Comment by phoenixy1** [comment]
@rajeevmehrotra I don't see your updates -- can you re-publish (I think you might need to regenerate the link in the readme or at the very least go through the flow and then click the update link button)? Also, while you're at it, it would probably be a good idea to delete the Income / Employment folder that has the old income endpoints in it? It's a little confusing that they're both in there.

**Review by phoenixy1 — APPROVED** [review]

**Pull request merged**